### DISCUSSION TITLE
How to access a "Scripting-Language" UDP synched global variable in Berry?

### DISCUSSION THREAD

**Discussion by HaraldNoack** [arendst/Tasmota#17633] [Q&A]
Hello,

I want to communicate between 2 ESP32 via UDP synched global variables ("Scripting-Language").

Example: 
ESP32 measures pressure in the second floor (Master) and must control a 4pin-fan that is  connected to a ESP32 in the ground floor (Slave).
The "Master" sets the PWM-value-variable and the "Slave" measures the RPM and sets the RPM-value-variable.
The controlling of the fan is a little bit complex, because there are special cases, like "window is opened/closed", PID anti windup and so on. So I wanted to write this in Berry.
But how can I access "Scripting-Language" global variables in Berry?
Or are there UDP synched global variable in Berry?

I don't want to use mqtt for this because there is no broker for this case.

"Scripting-Language"-Example
>D
g:PWM_value=0
g:RPM_value=0

>T
PWM_value=10

>W
PWM{m}%PWM_value% digit
Drehzahl{m}%RPM_value% RPM

Is it possible to write to "PWM_value" from berry?

THANKS foro your HELP!!!
Best Regards
Harald

**Comment by sfromis** [comment]
Not tried that - I think that you'd be able to execute a `script` command to set a variable, as suggested here:
https://tasmota.github.io/docs/Scripting-Language/#console-commands

**Reply by HaraldNoack** [reply]
Thanks,

but this is  "Scripting-Language".
How can I execute this commands by Berry?

**Reply by sfromis** [reply]
You did read the docs? Any command available from console or whatever is also available from Berry. This is not a language combination issue at all.
https://tasmota.github.io/docs/Berry/#tasmota-object

**Reply by s-hadinger** [reply]
However you need to figure how to get a Scripting variable from a Tasmota command. Why not use Berry at first?
